### PR TITLE
add capacity planning guidance for openshift metrics

### PR DESCRIPTION
**Pull request by jeremyeder**

### CODE REVIEW

**Comment by jeremyeder** [comment]
@ekuric @bfallonf @adellape

**Comment by bfallonf** [comment]
@jeremyeder @sdodson @adellape Does this tie into this BZ: 

https://bugzilla.redhat.com/show_bug.cgi?id=1303505

Alex, I followed the paper trail from your comment in #2682 .

Scott gave me the following link to document that BZ:

https://github.com/openshift/openshift-ansible/blob/master/inventory/byo/hosts.ose.example#L324-L365

Basically, I'm confused and don't see the link between the two and I don't mind documenting it all (or helping document of course) but I want to get my story straight before I do...

**Comment by jeremyeder** [comment]
I don't think it ties directly into that BZ since that's really just about how to use the new playbooks, right?  What this PR is about is augmenting the existing cluster_metrics docs with more information about capacity planning.

**Comment by adellape** [comment]
@jeremyeder Yeah, doesn't tie in directly. @bfallonf, I was mostly just pointing it out to bring awareness to these upcoming changes in this topic in case you'd be working in there too per [BZ#1303505](https://bugzilla.redhat.com/show_bug.cgi?id=1303505), or if you found that you wanted to reference the new material in your work.

File: install_config/cluster_metrics.adoc [hunk 1]
```diff
@@ -188,6 +247,8 @@ xref:../architecture/additional_concepts/storage.adoc#persistent-volume-claims[p
 volume claims] is handled by the
 xref:../install_config/cluster_metrics.adoc#metrics-deployer[Metrics Deployer].
 
+{product-title} metrics supports also https://github.com/openshift/origin-metrics/blob/master/metrics.yaml#L130[dynamically provisioned persistent volumes].  To use this feature with {product-title} metrics, it is necessary to add an additional flag to the metrics-deployer: `DYNAMICALLY_PROVISION_STORAGE=true`.  At this time, EBS, GCE and Cinder storage backends can be used in order to dynamically provision volumes.  For more information about Dynamically Provisioned PVs, refer to https://docs.openshift.com/enterprise/3.2/install_config/persistent_storage/dynamically_provisioning_pvs.html[Dynamically Provisioned PVS].
```
Comment (timothysc) [inline install_config/cluster_metrics.adoc]: that whole bit I mentioned on the other PR about non-versioned self references applies.

**Comment by ahardin-rh** [comment]
@jeremyeder Thanks! I am going to fork this and take it from here in a separate PR to get this moving forward.

**Comment by ahardin-rh** [comment]
closing in favor of https://github.com/openshift/openshift-docs/pull/2821

**Pull request closed without merging**